dashboard/app/models/game.rb

```diff
@@ -181,6 +185,7 @@ def self.setup
         Pixelation:pixelation
         TextCompression:text_compression
         Odometer:odometer
+        Vigenere:vigenere
```

Comment:
Since the `games` table gets recreated on each seed, adding `Vigenere` here will give an ID of 47, and push the `FrequencyAnalysis` game to ID 48.  Ask me how I know this :smile:.  (I did it wrong previously and had to go fix it.)